pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -261,10 +262,56 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, errors.Wrap(err, "failed to save cluster config")
 	}
 
+	poolAndRadosNamespaceName := fmt.Sprintf("%s/%s", pool, getRadosNamespaceName(cephBlockPoolRadosNamespace))
+	mirrorInfo, err := cephclient.GetPoolMirroringInfo(r.context, r.clusterInfo, poolAndRadosNamespaceName)
+	if err != nil {
+		return reconcile.Result{}, errors.Wrapf(err, "failed to get mirroring info for the radosnamespace %q", poolAndRadosNamespaceName)
+	}
+
+	if cephBlockPoolRadosNamespace.Spec.Mirroring.Enabled && mirrorInfo.Mode == "disabled" {
```

Comment:
why we need `mirrorInfo.Mode == "disabled"` check here? ideally, we don't need to depend on it and try to enable mirroring irrespective of it that's what we do for pool mirroring as well. we can avoid extra CLI call, What about the case where mirroring was enabled but we didn't add the scheduling for it (because for some reason operator got restart)